graylog2-server/src/main/java/org/graylog2/plugin/BaseConfiguration.java

```diff
@@ -163,6 +163,10 @@
     @Parameter(value = "web_tls_key_password")
     private String webTlsKeyPassword;
 
+    @Parameter(value = "proxied_requests_max_threads", required = true, validator = PositiveIntegerValidator.class)
+    // TODO: this is a totally abitrary number. this needs a better default based on ... something.
+    private int proxiedRequestsMaxThreads = 64;
```

Comment:
I agree with @bernd (https://github.com/Graylog2/graylog2-server/pull/2779#issuecomment-245103129) that a pool size of 64 threads is a bit too much for most setups.

8 or 16 threads should be fine for most workloads.